packages/code-studio/src/settings/FormattingSectionContent.tsx

```diff
@@ -596,6 +613,23 @@ export class FormattingSectionContent extends PureComponent<
               </Checkbox>
             </div>
           </div>
+
+          <div className="form-row mb-3">
+            <label
+              className="col-form-label col-3"
+              htmlFor="default-integer-format-input"
```

Comment:
This needs to be a different id (it's duplicated from above) and there needs to be a corresponding id on the Checkbox. Also we should just be using `for` probably since `htmlFor` is only needed if you need to modify the label ID w/ JS, but it doesn't really matter here

https://developer.mozilla.org/en-US/docs/Web/API/HTMLLabelElement/htmlFor

Reply:
I tried adding for, but then I have to add an exception since I am getting an ESLint error?